exclusion_mz.R

```diff
@@ -0,0 +1,62 @@
+NEG = c(
+  # Formic Acid:
+  44.9820, (46.00547*2)-1.007276, (44.9820 + seq(14)*67.98745),
+  (44.9820+(c(3,4)*67.98745)+57.95865),
+  758.86518, 758.86518+(seq(3)*67.98745),
+  
+  # RT 30 (IS)
+  61.98875, 115.9206, 116.92741, 132.9233, 144.9235, 
+  383.18958, 384.18454, 457.22673, 540.2722, 540.2722+1.00335,
+  
+  # RT 30 (MIX-01)
+  248.97233,
+  
+  # RT 110 (IS)
+  277.18426,
+  
+  # RT 122 (IS)
+  311.16875, 325.18432, 339.19983, 
+  
+  # RT 178 (IS)
+  195.8112, 197.8083, 199.80514,
+  
+  # ??
+  623.3915, (623.3915+seq(4)*67.98745)
+  )
+
+POS = c(124.0868, 90.97639, 90.97639+(seq(13)*67.98745),
+        
+        # RT 26 (IS)
+        223.0595, 283.0262,
+        # RT 27 (IS)
+        581.2343, 637.2957, (637.2957+1.00335), (637.2957+21.98198),
+        # RT 30 (IS)
+        341.2999,
+        # RT 31 (IS)
+        193.1589, 437.1869,
+        # RT 33 (IS)
+        455.3272,
+        # RT 40 (MIX-01)
+        60.04581,
+        # RT 45 (MIX-01)
+        74.06112,
+        # RT 74 (MIX-01)
+        114.0662, (114.0662+21.98198),
+        # RT 121 (IS)
+        135.09299,
+        # RT 196 (IS)
+        90.50735, 184.9847,
+        # RT 199 (IS)
+        89.50731, 113.96138, 141.95797,
+        # RT 203 (IS)
+        182.9841
+        )
+
+max.len = max(length(NEG), length(POS))
+
+NEG = c(NEG, rep(NA, max.len - length(NEG)))
+POS = c(POS, rep(NA, max.len - length(POS)))
+
+exclusion_mz <- data.frame(NEG, POS)
+
+write.table(exclusion_mz, "data/exclusion_mz.txt", sep="\t", row.names = FALSE)
```

Comment:
would be eventually interesting to add also a column `"notes"` that could contain the information from the comments (such as `"Formic acid"`, `"RT 203 (IS)"` etc.

Reply:
I've changed the structure of the code.
Let me know what do you think.